.travis.yml

```diff
@@ -0,0 +1,10 @@
+language: node_js
+
+node_js:
+  - stable
+
+install:
+  - npm install
+
+notification:
+  - email: false
```

Comment:
Why did disable email notifications?

Reply:
to avoid a lot of email notifications whenever a build is done

Reply:
so what if a build fails?

Reply:
If the build fails, you will immediately see the build badge fail, in red color

Reply:
ok cool